src/sentry/auth/idpmigration.py

```diff
@@ -1,6 +1,35 @@
-import uuid
+import string
+from datetime import timedelta
 
-from sentry.models import Organization, User
+from django.utils.crypto import get_random_string
+
+from sentry import options
+from sentry.models import Organization, OrganizationMember, User
+from sentry.utils import redis
+from sentry.utils.email import MessageBuilder
+
+_REDIS_KEY = "verificationKeyStorage"
```

Comment:
nit: do we need the `_` before the constants here? doesn't seem to be a standard across our codebase to mark module level variables this way, and IMO harms readability a tiny bit